frontend/js/utils/Utils.js

```diff
@@ -126,6 +126,8 @@ var Utils = {
   },
 
   userDisplayStr: function(userInfo) {
+    if(userInfo === null) return "";
```

Comment:
Noted, as a non-critical thing to fix later.

On Wed, Jan 6, 2016 at 2:44 AM, Aaron Zhou <notifications@github.com> wrote:

> In frontend/js/utils/Utils.js
> <https://github.com/lightvector/arimaa-server/pull/86#discussion_r48934023>
> :
>
> > @@ -126,6 +126,8 @@ var Utils = {
> >    },
> >
> >    userDisplayStr: function(userInfo) {
> > +    if(userInfo === null) return "";
>
> It happens if you try to go to a game that doesn't exist. We need a
> different "game not found" page for this case.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightvector/arimaa-server/pull/86/files#r48934023>.
>